crowdstrike/table_crowdstrike_user_role.go

```diff
@@ -0,0 +1,104 @@
+package crowdstrike
+
+import (
+	"context"
+
+	"github.com/crowdstrike/gofalcon/falcon"
+	"github.com/crowdstrike/gofalcon/falcon/client/user_management"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableCrowdStrikeUserRole(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "crowdstrike_user_role",
+		Description: "TODO.",
+		List: &plugin.ListConfig{
+			Hydrate: listCrowdStrikeUserRole,
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getCrowdStrikeUserRole,
+			KeyColumns: plugin.AllColumns([]string{"id"}),
+		},
+		Columns: []*plugin.Column{
+			{Name: "id", Description: "TODO", Type: proto.ColumnType_STRING, Hydrate: getCrowdStrikeUserRole},
+			{Name: "cid", Description: "TODO.", Type: proto.ColumnType_STRING, Hydrate: getCrowdStrikeUserRole},
+			{Name: "description", Description: "TODO.", Type: proto.ColumnType_STRING, Hydrate: getCrowdStrikeUserRole},
+			{Name: "display_name", Description: "TODO.", Type: proto.ColumnType_STRING, Hydrate: getCrowdStrikeUserRole},
+			// Steampipe standard columns
+			{Name: "title", Description: "Title of the resource.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Cid")},
```

Comment:
```suggestion
			{Name: "title", Description: "Title of the resource.", Type: proto.ColumnType_STRING, Transform: transform.FromField("DisplayName")},
```

Not sure if this is the exact field name, but display name seems better